modules/perception/camera/app/BUILD_

```diff
@@ -15,42 +15,13 @@ cc_library(
     ],
 )
 
```

Comment:
So, can this BUILD file be re-enabled?

Reply:
Some function need to be changed, so it will re-enabled in next commit.